TideApp/TideApp/Domain/Model/WeatherResponse.swift

```diff
@@ -64,6 +66,22 @@ struct WeatherData: Decodable {
           case tideDateTime
           case tideType = "tide_type"
         }
+        
+        init(from decoder: Decoder) throws {
```

Comment:
If you put this in an extension we still get access to the structs member wise initialiser without having to declare it again below.

Reply:
Thanks! I forgot about that